### PR TITLE
Moved Installation Kit and Logistic Kit so that they're part of Label Value Kit

### PR DESCRIPTION
**Pull request by christinaatai**
#### Screens

Screen Shot 2020-07-23 at 1 27 24 PM
Screen Shot 2020-07-22 at 10 05 29 AM
Screen Shot 2020-07-22 at 10 05 37 AM
Screen Shot 2020-07-22 at 10 07 47 AM
Screen Shot 2020-07-22 at 10 07 52 AM

#### Breaking Changes

This PR includes the removal of Logistic Kit and Installer Kit. They are not being used in Nitro web, so it _should_ be fine.

#### Runway Ticket URL

https://nitro.powerhrg.com/runway/backlog_items/NUX-1189

#### Checklist:

- [x] **DEPLOY** Please add the `Milano` label when you are ready for a review
- [x] **SCREENSHOT** Please add a screen shot or two
- [x] **SPECS** Please cover your changes with specs
- [x] **CHANGELOG** Please add an entry on `[Unreleased]` section to every functionality `ADDED`/`CHANGED`/`DEPRECATED`/`REMOVED`/`FIXED`

### CODE REVIEW

File: app/pb_kits/playbook/pb_label_value/_label_value.scss [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+//   }
+//   &[class*=_dark]  {
+//     [class^=pb_caption_kit] {
+//       color: $text_dk_light;
+//     }
+//     [class^=pb_body_kit] {
+//       color: $text_dk_default;
+//     }
+//   }
+// }
```
Comment (jasoncypret) [inline app/pb_kits/playbook/pb_label_value/_label_value.scss:17]: Can we remove instead?

File: app/pb_kits/playbook/pb_label_value/docs/_label_value_details.html.erb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  description: "33-12345",
+  title: "Jefferson-Smith",
+  date: Date.new(2019, 11, 18)
+}) %>
+
+<br>
+
+<%= pb_rails("label_value", props: {
+  variant: "details",
+  active: true,
```
Comment (jasoncypret) [inline app/pb_kits/playbook/pb_label_value/docs/_label_value_details.html.erb:33]: Can we make a short write up for all our kits on how to use? I'm not sure on this one, and I think writing a quick doc will help define the logic (when to use what)

File: app/pb_kits/playbook/pb_title/title.rb [hunk 3]
```diff
@@ -17,8 +17,8 @@ class Title
                  default: "h3"
       prop :text
       prop :variant, type: Playbook::Props::Enum,
-                    values: [nil, "primary"],
-                    default: nil
+                     values: [nil, "primary"],
```
Comment (jasoncypret) [inline app/pb_kits/playbook/pb_title/title.rb:20]: I wonder if this make sense in terms of other text color styles? For example, I believe they are called positive, negative, and link in sketch. Thoughts?

File: app/pb_kits/playbook/pb_label_value/docs/_label_value_details.html.erb (hunk 2 above)
Comment (christinaatai) [reply app/pb_kits/playbook/pb_label_value/docs/_label_value_details.html.erb:33]: I updated the description of the kit.

Screen Shot 2020-07-23 at 1 27 24 PM

**Comment by jasoncypret** [comment]
@christinaatai are you planning this for 5.6.0 release

**Pull request merged**